e2e/framework/framework.go

```diff
@@ -298,7 +298,7 @@ func fetchOpsCenterLogs() error {
 		return trace.Wrap(err, "failed to collect site report")
 	}
 
-	err = system.CopyFile(output, tempOutput.Name())
+	err = system.CopyFile(tempOutput.Name(), output)
```

Comment:
stdlib is counter intuitive by itself... when you do cp its always source, then destination